utils/utils.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import "strings"
+
+// NormalizeService normalizes a provided service to a common format
+func NormalizeService(service string) string {
+	return strings.ToLower(service)
+}
+
+// NormalizeKey normalizes a provided secret key to a common format
+func NormalizeKey(key string) string {
+	return strings.ToLower(key)
+}
```

Comment:
What's the intent in moving these to a separate package? Since these both just call `strings.ToLower()`, right now it's just indirection on top of stdlib.

Reply:
just standardization/normalization. There are separate packages in this repo that rely on the same logic, so was attempting to DRY it out by giving it an interface

Reply:
🤷🏻‍♂️ I'd argue that 2 new methods that both lowercase a string is a worse deal than just using `strings.ToLower()` when needed buuuuuuut it's not a hill I care about dying on.

Reply:
Totally fair - I might be overthinking the need to update how or what normalization means in the future and implemented this as a measure.

I may have also thought that the lack of a normalizing function (and just raw `strings.ToLower`) may have results in missed normalization in the past, which lead me to move towards it

Reply:
Yeah, I can see that; if any of the other normalizion/sanitization happened in these functions, I think it'd make more sense to have them. As they're written right now, they're just 1:1 wrappers around `strings.ToLower()` without any benefit for the indirection.